eva-accession-import-dbsnp2/src/test/resources/input-files/assembly-report/GCF_000001405.38_GRCh38.p12_assembly_report.txt

```diff
@@ -0,0 +1,640 @@
+# Assembly name:  GRCh38.p12

```

Comment:
If the vast majority of the lines in the assembly report are not used, could they be cut off of the file?

Reply:
Because the file isn't too huge, I favour to bundle this one with the code. Otherwise, the unused lines can be cropped